statsmodels/kernel_methods/_kernels1d.py

```diff
@@ -0,0 +1,288 @@
+from .kernels import Kernel1D, rfftfreq, dctfreq
+from . import _cy_kernels
+import numpy as np
+
+
+class tricube(Kernel1D):
```

Comment:
capitalize class names

Reply:
Yes definitely, I'll do this when PEP8-ing everything